backend/golf/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
```

Comment:
This should be part of a separate PR.  Try to group them by functionality, and break them into as small of pieces as possible.  Lots of tiny reviews > big reviews